packages/manager/apps/vrack-services/mock/setup.ts

```diff
@@ -6,7 +6,7 @@ export const setupMocks = async () =>
   setupWorker(
     ...toMswHandlers(
       getConfig({
-        nbVs: 19,
+        nbVs: 20,
```

Comment:
```suggestion
        nbVs: 19,
```

It's the setup used in dev mode but if we put 20, the create button is disabled

Reply:
oh Ok I will change that